src/Mod/Arch/importWebGL.py

```diff
@@ -18,233 +19,782 @@
 #*   USA                                                                   *
 #*                                                                         *
 #***************************************************************************
+#
+# REFS:
+# https://github.com/mrdoob/three.js/blob/master/examples/webgl_interactive_buffergeometry.html
+# https://threejs.org/examples/#webgl_buffergeometry_lines
+# https://forum.freecadweb.org/viewtopic.php?t=51245
+# https://forum.freecadweb.org/viewtopic.php?t=29487
+# https://threejs.org/examples/#webgl_raycast_sprite
+#
+# Params for export()
+#    'colors' is of the form: {'Body': [1,0,0], 'Body001': [1,1,0], 'Body002': [1,0,1] }
+#    'camera' is of the form: "PerspectiveCamera {\n  viewportMapping ADJUST_CAMERA\n  position 30.242626 -51.772324 85.63475\n  orientation -0.4146691 0.088459305 -0.90566254  4.7065201\nnearDistance 53.126431\n  farDistance 123.09125\n  aspectRatio 1\n  focalDistance 104.53851\n  heightAngle 0.78539819\n\n}"
+#    The 'camera' string for the active document may be generated from: import OfflineRenderingUtils; OfflineRenderingUtils.getCamera(FreeCAD.ActiveDocument.FileName);
+#
+# Development reload oneliner:
+# def re(): from importlib import reload;import importWebGL;reload(importWebGL);o=FreeCAD.getDocument("curve");importWebGL.export([o.getObject("Body")],u"C:/Users/Travis/Desktop/test.htm");
```

Comment:
`C:/Users/Travis/Desktop/test.htm"` is hardcoded

Reply:
That's just an example for the next dev. It took me a while to find a quick way to test this code and so I put that in there to save a step for them.

Reply:
But you're right. I've replaced the names with something more descriptive